tests/test_packages/test_connections/test_ledger_api/test_ledger_api.py

```diff
@@ -90,23 +97,45 @@ async def test_get_balance(ledger_id, address, ledger_apis_connection: Connectio
     assert actual_balance_amount == expected_balance_amount
 
 
-# @pytest.mark.asyncio
-# @ledger_ids
-# async def test_send_signed_transaction(ledger_id, address, ledger_apis_connection: Connection):
-#     """Test send signed transaction."""
-#     request = LedgerApiMessage(
-#         LedgerApiMessage.Performative.SEND_SIGNED_TRANSACTION,
-#         ledger_id=ledger_id,
-#         address=address
-#     )
-#     envelope = Envelope("", "", request.protocol_id, message=request)
-#     await ledger_apis_connection.send(envelope)
-#     await asyncio.sleep(0.01)
-#     response = await ledger_apis_connection.receive()
-#
-#     assert response is not None
-#     assert type(response.message) == LedgerApiMessage
-#     message = cast(LedgerApiMessage, response.message)
+@pytest.mark.asyncio
+async def test_send_signed_transaction_ethereum(ledger_apis_connection: Connection):
+    """Test send signed transaction with Ethereum APIs."""
+    crypto1 = EthereumCrypto(private_key_path=ETHEREUM_PRIVATE_KEY_PATH)
+    crypto2 = EthereumCrypto()
+    api = aea.crypto.registries.make_ledger_api(EthereumCrypto.identifier)
+    api = cast(EthereumApi, api)
+
+    amount = 40000
+    fee = 30000
+    tx_nonce = api.generate_tx_nonce(crypto1.address, crypto2.address)
+    tx = make_ethereum_transaction(
```

Comment:
see below

Reply:
Not a good idea; we should use our wrappers instead imo